spec/models/merchant_spec.rb

```diff
@@ -84,7 +83,15 @@
 
     it "enable_disable" do
       expect(@meg.enable_disable).to eq(true)
-      expect(@meg.enable_disable).to eq(false)
+      @meg.enable_disable
+      expect(@meg.disabled).to eq(false)
+    end
+
+    it "deactivate_items" do
+      @meg.deactivate_items
+
+      expect(@tire.active?).to eq(false)
+      expect(@seat.active?).to eq(false)
```

Comment:
Update enable_disable test as well as add test for the new merchant model method